src/pages/Signup.js

```diff
@@ -0,0 +1,538 @@
+import React from 'react';
+import '../styles/common.scss';
+import '../pages/Signup.scss';
+
+class Signup extends React.Component {
+  constructor() {
+    super();
+    this.state = {
+      username: '',
+      password: '',
+      checkpassword: '',
+      name: '',
+      address1: '',
+      address2: '',
+      gender: '',
+    };
+  }
+
+  handleInput = e => {
+    const { name, value } = e.target;
+    this.setState({
+      [name]: value,
+    });
+
+    this.setState({ birthday: value }, () => this.sumBirthDay());
+
+    this.setState({ mobile_number: value }, () => this.sumMoblieNumber());
+
+    this.setState({ email: value }, () => this.sumEmail());
+  };
+
+  sumBirthDay = () => {
+    const { year, month, day } = this.state;
+    this.setState({ birthday: year + '-' + month + '-' + day });
+  };
+
+  sumMoblieNumber = () => {
+    const { first_number, second_number, last_number } = this.state;
+    this.setState({
+      mobile_number: first_number + second_number + last_number,
+    });
+  };
+
+  sumEmail = () => {
+    const { email_id, emailaddress } = this.state;
+    this.setState({
+      email: email_id + '@' + emailaddress,
+    });
+  };
+
+  goToSignin = () => {
+    const {
+      username,
+      password,
+      name,
+      email,
+      mobile_number,
+      address1,
+      address2,
+      birthday,
+      gender,
+    } = this.state;
+    this.props.history.push('/users/signin');
+    fetch('http://10.58.7.38:8000/users/signup', {
+      method: 'POST',
+      body: JSON.stringify({
+        username: username,
+        password: password,
+        name: name,
+        email: email,
+        mobile_number: mobile_number,
+        address1: address1,
+        address2: address2,
+        birthday: birthday,
+        gender: gender,
+      }),
+    })
+      .then(response => response.json())
+      .then(result => console.log(''));
+  };
```

Comment:
```suggestion
    const {
      username,
      password,
      name,
      email,
      mobile_number,
      address1,
      address2,
      birthday,
      gender,
    } = this.state;
    
    this.props.history.push('/users/signin');
    
    fetch('http://10.58.7.38:8000/users/signup', {
      method: 'POST',
      body: JSON.stringify({
        username: username,
        password: password,
        name: name,
        email: email,
        mobile_number: mobile_number,
        address1: address1,
        address2: address2,
        birthday: birthday,
        gender: gender,
      }),
    })
      .then(response => response.json())
      .then(result => console.log(''));
  };
```
다른 로직마다 공백 주는게 가독성에 좋고 주소부분은 저번에 코멘트드린 config.js 적용해보세요!

Reply:
넵 수정했습니다~!!!